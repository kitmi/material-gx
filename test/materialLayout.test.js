import React from 'react';
import renderer from 'react-test-renderer';
import { materialLayoutCreator } from '../build';

const MaterialLayout = materialLayoutCreator(240, 80);

it('renders correctly', () => {
    const tree = renderer
        .create(<MaterialLayout sidebar={<div>sidebar</div>} content={<div>content</div>} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});