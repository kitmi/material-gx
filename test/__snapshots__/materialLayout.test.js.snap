// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Material-root-1"
>
  <div>
    <div
      className="MuiDrawer-docked-9"
      onMouseOut={[Function]}
      onMouseOver={[Function]}
    >
      <div
        className="MuiPaper-root-20 MuiPaper-elevation0-22 MuiDrawer-paper-10 Material-drawerPaper-2 MuiDrawer-paperAnchorLeft-11 MuiDrawer-paperAnchorDockedLeft-15"
      >
        <div>
          sidebar
        </div>
      </div>
    </div>
  </div>
  <main
    className="Material-mainPanel-4"
  >
    <div
      className="Material-content-6"
    >
      <div
        className="Material-container-7"
      >
        <div>
          content
        </div>
      </div>
    </div>
  </main>
</div>
`;
